src/domains/core/workspace/workspace.container.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { Redirect } from 'react-router-dom';
 
 import { LOGIN_PATH } from 'routes';
+import { fetchFavoritesSongs } from 'store/actions/favoritesActions';
 import { WorkspaceComponent } from './workspace.component';
 import { selectIsSessionToken } from '../app.selectors';
 
 export const Workspace = () => {
   const isSessionToken = useSelector(selectIsSessionToken);
+  const dispatch = useDispatch();
+
+  if (isSessionToken) {
+    dispatch(fetchFavoritesSongs());
+  }
```

Comment:
This is not good. Please take a look at this:

https://reactjs.org/docs/hooks-reference.html